src/components/AddBook/AddBook.js

```diff
@@ -63,11 +52,7 @@ const AddBook = () => {
       <button
         type="submit"
         onClick={submitBookHandler}
-        disabled={
-          bookTitle.trim() === '' ||
-          bookAuthor.trim() === '' ||
-          categoryValue === 'none'
-        }
+        disabled={bookTitle.trim() === '' || categoryValue === 'none'}
       >
         Add
       </button>
```

Comment:
- [ ] Let us implement this when a user adds the book the entered data kept in the user input fields. So let us make sure that when a user sends data, the input has to be empty so that a user can add new data as well.

![cc-1](https://user-images.githubusercontent.com/55635977/150200422-a35b238b-6bdf-4816-ab99-23d57d4bdb1d.png)